Need help with enabling libssh

I made 50-libssh.sh similar to other libraries
https://github.com/megapro17/FFmpeg-Builds/blob/master/scripts.d/50-libssh.sh
It did compiled fine, but when building ffmpeg it says
`ERROR: libssh not found using pkg-config`
I've tried googling this mistake, but didn't found much info
https://github.com/megapro17/FFmpeg-Builds/runs/6779032432?check_suite_focus=true

libssh is adding support for reading and writing files from sftp server, which is very useful, because you can run encoding on remote machines without manual file transfers

I've found a correct options here , now it builds, and almost in the end new error appears... 
https://github.com/wader/static-ffmpeg/blob/6048294be4fb3dd75c97469eecbfa693ddb286a1/Dockerfile#L465
I've tried using that 0.9.6 version, it fails at pkg-config, with master it goes fine
It doesn't even appear in config.log
```
x86_64-w64-mingw32-gcc -Llibavcodec -Llibavdevice -Llibavfilter -Llibavformat -Llibavutil -Llibpostproc -Llibswscale -Llibswresample -static-libgcc -static-libstdc++ -L/opt/ffbuild/lib -O2 -pipe -fstack-protector-strong -pthread  -Wl,--nxcompat,--dynamicbase -Wl,--high-entropy-va -Wl,--as-needed -Wl,--warn-common -Wl,-rpath-link=:libpostproc:libswresample:libswscale:libavfilter:libavdevice:libavformat:libavcodec:libavutil  -Wl,--pic-executable,-e,mainCRTStartup -Wl,--image-base,0x140000000 -o ffmpeg_g.exe fftools/ffmpeg_filter.o fftools/ffmpeg_hw.o fftools/ffmpeg_mux.o fftools/ffmpeg_opt.o  fftools/cmdutils.o fftools/opt_common.o fftools/ffmpeg.o  -lavdevice -lavfilter -lavformat -lavcodec -lpostproc -lswresample -lswscale -lavutil  -lpsapi -lole32 -lstrmiids -luuid -loleaut32 -lshlwapi -lgdi32 -lm -latomic -lvfw32 -L/opt/ffbuild/lib -lSDL2 -lmingw32 -ldinput8 -lshell32 -lsetupapi -ladvapi32 -luuid -lversion -loleaut32 -lole32 -limm32 -lwinmm -lgdi32 -luser32 -lm -pthread -lm -latomic -L/opt/ffbuild/lib -llilv-0 -lm -lm -lsratom-0 -lm -lsord-0 -lserd-0 -lm -L/opt/ffbuild/lib -lrubberband -L/opt/ffbuild/lib -lfftw3 -lm -lsamplerate -lstdc++ -L/opt/ffbuild/lib -lfribidi -L/opt/ffbuild/lib -lplacebo -lm -lversion -lshaderc_combined -lspirv-cross-c -lspirv-cross-glsl -lspirv-cross-hlsl -lspirv-cross-reflect -lspirv-cross-msl -lspirv-cross-util -lspirv-cross-core -lstdc++ -L/opt/ffbuild/lib -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lass -liconv -lgdi32 -L/opt/ffbuild/lib -lfontconfig -L/opt/ffbuild/lib -lfreetype -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -lharfbuzz -lm -L/opt/ffbuild/lib -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lfribidi -lfreetype -lz -L/opt/ffbuild/lib -lshaderc_combined -lstdc++ -L/opt/ffbuild/lib -lvidstab -lm -lgomp -lpthread -L/opt/ffbuild/lib -lzimg -lstdc++ -L/opt/ffbuild/lib -lOpenCL -lole32 -lshlwapi -lcfgmgr32 -L/opt/ffbuild/lib -lfontconfig -lfreetype -lz -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -L/opt/ffbuild/lib -lfreetype -lz -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -lm -latomic -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -L/opt/ffbuild/lib -lgme -lstdc++ -lz -L/opt/ffbuild/lib -lopenmpt -lole32 -lrpcrt4 -lstdc++ -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -logg -lvorbis -lm -L/opt/ffbuild/lib -logg -lvorbisfile -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -lstdc++ -L/opt/ffbuild/lib -lbluray -lgdi32 -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -lfreetype -lz -ludfread -lgmp -L/opt/ffbuild/lib -lz -lsecur32 -L/opt/ffbuild/lib -lrist -lws2_32 -pthread -lmbedcrypto -L/opt/ffbuild/lib -lsrt -lwsock32 -lws2_32 -lstdc++ -lssp_nonshared -lssp -lmingw32 -lgcc -lmoldname -lmingwex -lkernel32 -lpthread -ladvapi32 -lshell32 -luser32 -lkernel32 -lmingw32 -lgcc -lmoldname -lmingwex -lkernel32 -L/opt/ffbuild/lib -lssl -L/opt/ffbuild/lib -lz -lws2_32 -lgdi32 -lcrypt32 -lz -lws2_32 -lgdi32 -lcrypt32 -lcrypto -lz -lws2_32 -lgdi32 -lcrypt32 -lws2_32 -L/opt/ffbuild/lib -lssh -DLIBSSH_STATIC=1 -lssh -L/opt/ffbuild/lib -lssl -L/opt/ffbuild/lib -lz -lws2_32 -lgdi32 -lcrypt32 -lcrypto -lws2_32 -lgdi32 -lcrypt32 -lz -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lwebpmux -lm -lwebp -lm -pthread -liconv -lm -latomic -llzma -L/opt/ffbuild/lib -laribb24 -lm -lpng16 -lz -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -ldav1d -L/opt/ffbuild/lib -ldavs2 -lopencore-amrwb -L/opt/ffbuild/lib -luavs3d -lm -lpthread -L/opt/ffbuild/lib -lzvbi -lpthread -lm -liconv -L/opt/ffbuild/lib -laom -lpthread -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lfdk-aac -L/opt/ffbuild/lib -ljxl -lm -lstdc++ -ladvapi32 -L/opt/ffbuild/lib -lhwy -lbrotlicommon -lbrotlienc -L/opt/ffbuild/lib -lbrotlicommon -lbrotlidec -L/opt/ffbuild/lib -lbrotlicommon -L/opt/ffbuild/lib -ljxl_threads -lm -lstdc++ -ladvapi32 -lmp3lame -lm -lopencore-amrnb -L/opt/ffbuild/lib -lopenjp2 -lm -L/opt/ffbuild/lib -lopus -lm -L/opt/ffbuild/lib -lrav1e -lws2_32 -lbcrypt -ladvapi32 -luserenv -lkernel32 -lkernel32 -lws2_32 -lbcrypt -ladvapi32 -luserenv -lkernel32 -L/opt/ffbuild/lib -lSvtAv1Enc -ltheoraenc -ltheoradec -logg -ltwolame -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -L/opt/ffbuild/lib -lvorbisenc -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -L/opt/ffbuild/lib -lwebp -lm -L/opt/ffbuild/lib -lx264 -L/opt/ffbuild/lib -lx265 -lstdc++ -lssp_nonshared -lssp -lgcc -lgcc -L/opt/ffbuild/lib -lxavs2 -lxvidcore -L/opt/ffbuild/lib -lopenh264 -lstdc++ -lmfuuid -lole32 -lstrmiids -lole32 -luser32 -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -lm -latomic -lm -lsoxr -latomic -lm -latomic -pthread -lm -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -L/opt/ffbuild/lib -lOpenCL -lole32 -lshlwapi -lcfgmgr32 -luser32 -lbcrypt -latomic -lgomp  -lole32 -lpsapi -lshell32
x86_64-w64-mingw32-gcc -Llibavcodec -Llibavdevice -Llibavfilter -Llibavformat -Llibavutil -Llibpostproc -Llibswscale -Llibswresample -static-libgcc -static-libstdc++ -L/opt/ffbuild/lib -O2 -pipe -fstack-protector-strong -pthread  -Wl,--nxcompat,--dynamicbase -Wl,--high-entropy-va -Wl,--as-needed -Wl,--warn-common -Wl,-rpath-link=:libpostproc:libswresample:libswscale:libavfilter:libavdevice:libavformat:libavcodec:libavutil  -Wl,--pic-executable,-e,mainCRTStartup -Wl,--image-base,0x140000000 -o ffplay_g.exe fftools/cmdutils.o fftools/opt_common.o fftools/ffplay.o  -lavdevice -lavfilter -lavformat -lavcodec -lpostproc -lswresample -lswscale -lavutil  -lpsapi -lole32 -lstrmiids -luuid -loleaut32 -lshlwapi -lgdi32 -lm -latomic -lvfw32 -L/opt/ffbuild/lib -lSDL2 -lmingw32 -ldinput8 -lshell32 -lsetupapi -ladvapi32 -luuid -lversion -loleaut32 -lole32 -limm32 -lwinmm -lgdi32 -luser32 -lm -pthread -lm -latomic -L/opt/ffbuild/lib -llilv-0 -lm -lm -lsratom-0 -lm -lsord-0 -lserd-0 -lm -L/opt/ffbuild/lib -lrubberband -L/opt/ffbuild/lib -lfftw3 -lm -lsamplerate -lstdc++ -L/opt/ffbuild/lib -lfribidi -L/opt/ffbuild/lib -lplacebo -lm -lversion -lshaderc_combined -lspirv-cross-c -lspirv-cross-glsl -lspirv-cross-hlsl -lspirv-cross-reflect -lspirv-cross-msl -lspirv-cross-util -lspirv-cross-core -lstdc++ -L/opt/ffbuild/lib -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lass -liconv -lgdi32 -L/opt/ffbuild/lib -lfontconfig -L/opt/ffbuild/lib -lfreetype -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -lharfbuzz -lm -L/opt/ffbuild/lib -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lfribidi -lfreetype -lz -L/opt/ffbuild/lib -lshaderc_combined -lstdc++ -L/opt/ffbuild/lib -lvidstab -lm -lgomp -lpthread -L/opt/ffbuild/lib -lzimg -lstdc++ -L/opt/ffbuild/lib -lOpenCL -lole32 -lshlwapi -lcfgmgr32 -L/opt/ffbuild/lib -lfontconfig -lfreetype -lz -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -L/opt/ffbuild/lib -lfreetype -lz -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -lm -latomic -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -L/opt/ffbuild/lib -lgme -lstdc++ -lz -L/opt/ffbuild/lib -lopenmpt -lole32 -lrpcrt4 -lstdc++ -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -logg -lvorbis -lm -L/opt/ffbuild/lib -logg -lvorbisfile -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -lstdc++ -L/opt/ffbuild/lib -lbluray -lgdi32 -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -lfreetype -lz -ludfread -lgmp -L/opt/ffbuild/lib -lz -lsecur32 -L/opt/ffbuild/lib -lrist -lws2_32 -pthread -lmbedcrypto -L/opt/ffbuild/lib -lsrt -lwsock32 -lws2_32 -lstdc++ -lssp_nonshared -lssp -lmingw32 -lgcc -lmoldname -lmingwex -lkernel32 -lpthread -ladvapi32 -lshell32 -luser32 -lkernel32 -lmingw32 -lgcc -lmoldname -lmingwex -lkernel32 -L/opt/ffbuild/lib -lssl -L/opt/ffbuild/lib -lz -lws2_32 -lgdi32 -lcrypt32 -lz -lws2_32 -lgdi32 -lcrypt32 -lcrypto -lz -lws2_32 -lgdi32 -lcrypt32 -lws2_32 -L/opt/ffbuild/lib -lssh -DLIBSSH_STATIC=1 -lssh -L/opt/ffbuild/lib -lssl -L/opt/ffbuild/lib -lz -lws2_32 -lgdi32 -lcrypt32 -lcrypto -lws2_32 -lgdi32 -lcrypt32 -lz -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lwebpmux -lm -lwebp -lm -pthread -liconv -lm -latomic -llzma -L/opt/ffbuild/lib -laribb24 -lm -lpng16 -lz -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -ldav1d -L/opt/ffbuild/lib -ldavs2 -lopencore-amrwb -L/opt/ffbuild/lib -luavs3d -lm -lpthread -L/opt/ffbuild/lib -lzvbi -lpthread -lm -liconv -L/opt/ffbuild/lib -laom -lpthread -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lfdk-aac -L/opt/ffbuild/lib -ljxl -lm -lstdc++ -ladvapi32 -L/opt/ffbuild/lib -lhwy -lbrotlicommon -lbrotlienc -L/opt/ffbuild/lib -lbrotlicommon -lbrotlidec -L/opt/ffbuild/lib -lbrotlicommon -L/opt/ffbuild/lib -ljxl_threads -lm -lstdc++ -ladvapi32 -lmp3lame -lm -lopencore-amrnb -L/opt/ffbuild/lib -lopenjp2 -lm -L/opt/ffbuild/lib -lopus -lm -L/opt/ffbuild/lib -lrav1e -lws2_32 -lbcrypt -ladvapi32 -luserenv -lkernel32 -lkernel32 -lws2_32 -lbcrypt -ladvapi32 -luserenv -lkernel32 -L/opt/ffbuild/lib -lSvtAv1Enc -ltheoraenc -ltheoradec -logg -ltwolame -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -L/opt/ffbuild/lib -lvorbisenc -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -L/opt/ffbuild/lib -lwebp -lm -L/opt/ffbuild/lib -lx264 -L/opt/ffbuild/lib -lx265 -lstdc++ -lssp_nonshared -lssp -lgcc -lgcc -L/opt/ffbuild/lib -lxavs2 -lxvidcore -L/opt/ffbuild/lib -lopenh264 -lstdc++ -lmfuuid -lole32 -lstrmiids -lole32 -luser32 -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -lm -latomic -lm -lsoxr -latomic -lm -latomic -pthread -lm -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -L/opt/ffbuild/lib -lOpenCL -lole32 -lshlwapi -lcfgmgr32 -luser32 -lbcrypt -latomic -lgomp  -L/opt/ffbuild/lib -lSDL2 -lmingw32 -ldinput8 -lshell32 -lsetupapi -ladvapi32 -luuid -lversion -loleaut32 -lole32 -limm32 -lwinmm -lgdi32 -luser32 -lm -lshell32x86_64-w64-mingw32-gcc -Llibavcodec -Llibavdevice -Llibavfilter -Llibavformat -Llibavutil -Llibpostproc -Llibswscale -Llibswresample -static-libgcc -static-libstdc++ -L/opt/ffbuild/lib -O2 -pipe -fstack-protector-strong -pthread  -Wl,--nxcompat,--dynamicbase -Wl,--high-entropy-va -Wl,--as-needed -Wl,--warn-common -Wl,-rpath-link=:libpostproc:libswresample:libswscale:libavfilter:libavdevice:libavformat:libavcodec:libavutil  -Wl,--pic-executable,-e,mainCRTStartup -Wl,--image-base,0x140000000 -o ffprobe_g.exe fftools/cmdutils.o fftools/opt_common.o fftools/ffprobe.o  -lavdevice -lavfilter -lavformat -lavcodec -lpostproc -lswresample -lswscale -lavutil  -lpsapi -lole32 -lstrmiids -luuid -loleaut32 -lshlwapi -lgdi32 -lm -latomic -lvfw32 -L/opt/ffbuild/lib -lSDL2 -lmingw32 -ldinput8 -lshell32 -lsetupapi -ladvapi32 -luuid -lversion -loleaut32 -lole32 -limm32 -lwinmm -lgdi32 -luser32 -lm -pthread -lm -latomic -L/opt/ffbuild/lib -llilv-0 -lm -lm -lsratom-0 -lm -lsord-0 -lserd-0 -lm -L/opt/ffbuild/lib -lrubberband -L/opt/ffbuild/lib -lfftw3 -lm -lsamplerate -lstdc++ -L/opt/ffbuild/lib -lfribidi -L/opt/ffbuild/lib -lplacebo -lm -lversion -lshaderc_combined -lspirv-cross-c -lspirv-cross-glsl -lspirv-cross-hlsl -lspirv-cross-reflect -lspirv-cross-msl -lspirv-cross-util -lspirv-cross-core -lstdc++ -L/opt/ffbuild/lib -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lass -liconv -lgdi32 -L/opt/ffbuild/lib -lfontconfig -L/opt/ffbuild/lib -lfreetype -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -lharfbuzz -lm -L/opt/ffbuild/lib -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lfribidi -lfreetype -lz -L/opt/ffbuild/lib -lshaderc_combined -lstdc++ -L/opt/ffbuild/lib -lvidstab -lm -lgomp -lpthread -L/opt/ffbuild/lib -lzimg -lstdc++ -L/opt/ffbuild/lib -lOpenCL -lole32 -lshlwapi -lcfgmgr32 -L/opt/ffbuild/lib -lfontconfig -lfreetype -lz -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -L/opt/ffbuild/lib -lfreetype -lz -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -lm -latomic -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -L/opt/ffbuild/lib -lgme -lstdc++ -lz -L/opt/ffbuild/lib -lopenmpt -lole32 -lrpcrt4 -lstdc++ -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -logg -lvorbis -lm -L/opt/ffbuild/lib -logg -lvorbisfile -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -lstdc++ -L/opt/ffbuild/lib -lbluray -lgdi32 -L/opt/ffbuild/lib -lxml2 -lm -lws2_32 -lfreetype -lz -ludfread -lgmp -L/opt/ffbuild/lib -lz -lsecur32 -L/opt/ffbuild/lib -lrist -lws2_32 -pthread -lmbedcrypto -L/opt/ffbuild/lib -lsrt -lwsock32 -lws2_32 -lstdc++ -lssp_nonshared -lssp -lmingw32 -lgcc -lmoldname -lmingwex -lkernel32 -lpthread -ladvapi32 -lshell32 -luser32 -lkernel32 -lmingw32 -lgcc -lmoldname -lmingwex -lkernel32 -L/opt/ffbuild/lib -lssl -L/opt/ffbuild/lib -lz -lws2_32 -lgdi32 -lcrypt32 -lz -lws2_32 -lgdi32 -lcrypt32 -lcrypto -lz -lws2_32 -lgdi32 -lcrypt32 -lws2_32 -L/opt/ffbuild/lib -lssh -DLIBSSH_STATIC=1 -lssh -L/opt/ffbuild/lib -lssl -L/opt/ffbuild/lib -lz -lws2_32 -lgdi32 -lcrypt32 -lcrypto -lws2_32 -lgdi32 -lcrypt32 -lz -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lvpx -lm -lm -lpthread -L/opt/ffbuild/lib -lwebpmux -lm -lwebp -lm -pthread -liconv -lm -latomic -llzma -L/opt/ffbuild/lib -laribb24 -lm -lpng16 -lz -L/opt/ffbuild/lib -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -ldav1d -L/opt/ffbuild/lib -ldavs2 -lopencore-amrwb -L/opt/ffbuild/lib -luavs3d -lm -lpthread -L/opt/ffbuild/lib -lzvbi -lpthread -lm -liconv -L/opt/ffbuild/lib -laom -lpthread -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lvmaf -pthread -lm -lstdc++ -L/opt/ffbuild/lib -lfdk-aac -L/opt/ffbuild/lib -ljxl -lm -lstdc++ -ladvapi32 -L/opt/ffbuild/lib -lhwy -lbrotlicommon -lbrotlienc -L/opt/ffbuild/lib -lbrotlicommon -lbrotlidec -L/opt/ffbuild/lib -lbrotlicommon -L/opt/ffbuild/lib -ljxl_threads -lm -lstdc++ -ladvapi32 -lmp3lame -lm -lopencore-amrnb -L/opt/ffbuild/lib -lopenjp2 -lm -L/opt/ffbuild/lib -lopus -lm -L/opt/ffbuild/lib -lrav1e -lws2_32 -lbcrypt -ladvapi32 -luserenv -lkernel32 -lkernel32 -lws2_32 -lbcrypt -ladvapi32 -luserenv -lkernel32 -L/opt/ffbuild/lib -lSvtAv1Enc -ltheoraenc -ltheoradec -logg -ltwolame -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -L/opt/ffbuild/lib -lvorbisenc -L/opt/ffbuild/lib -lvorbis -lm -L/opt/ffbuild/lib -logg -L/opt/ffbuild/lib -lwebp -lm -L/opt/ffbuild/lib -lx264 -L/opt/ffbuild/lib -lx265 -lstdc++ -lssp_nonshared -lssp -lgcc -lgcc -L/opt/ffbuild/lib -lxavs2 -lxvidcore -L/opt/ffbuild/lib -lopenh264 -lstdc++ -lmfuuid -lole32 -lstrmiids -lole32 -luser32 -L/opt/ffbuild/lib -lz -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -lm -latomic -lm -lsoxr -latomic -lm -latomic -pthread -lm -L/opt/ffbuild/lib -lmfx -lstdc++ -lole32 -luuid -L/opt/ffbuild/lib -lOpenCL -lole32 -lshlwapi -lcfgmgr32 -luser32 -lbcrypt -latomic -lgomp  -lshell32
/opt/ct-ng/lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld: /opt/ffbuild/lib/libssh.a(misc.c.obj):misc.c:(.text+0x1e0): multiple definition of `gettimeofday'; /opt/ct-ng/x86_64-w64-mingw32/sysroot/mingw/lib/../lib/libmingwex.a(lib64_libmingwex_a-gettimeofday.o):/ct-ng/build/x86_64-w64-mingw32/src/mingw-w64/mingw-w64-crt/misc/gettimeofday.c:69: first defined here/opt/ct-ng/lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld: /opt/ffbuild/lib/libssh.a(misc.c.obj):misc.c:(.text+0x1e0): multiple definition of `gettimeofday'; /opt/ct-ng/x86_64-w64-mingw32/sysroot/mingw/lib/../lib/libmingwex.a(lib64_libmingwex_a-gettimeofday.o):/ct-ng/build/x86_64-w64-mingw32/src/mingw-w64/mingw-w64-crt/misc/gettimeofday.c:69: first defined here/opt/ct-ng/lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld: /opt/ffbuild/lib/libssh.a(misc.c.obj):misc.c:(.text+0x1e0): multiple definition of `gettimeofday'; /opt/ct-ng/x86_64-w64-mingw32/sysroot/mingw/lib/../lib/libmingwex.a(lib64_libmingwex_a-gettimeofday.o):/ct-ng/build/x86_64-w64-mingw32/src/mingw-w64/mingw-w64-crt/misc/gettimeofday.c:69: first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:130: ffplay_g.exe] Error 1
make: *** Waiting for unfinished jobs....
collect2: error: ld returned 1 exit status
make: *** [Makefile:130: ffprobe_g.exe] Error 1
collect2: error: ld returned 1 exit status
make: *** [Makefile:130: ffmpeg_g.exe] Error 1
+ rm -f -- /tmp/tmp.sfDvZuBnXg
```

--- a/scripts.d/50-libssh.sh
+++ b/scripts.d/50-libssh.sh
@@ -13,26 +13,22 @@ ffbuild_dockerbuild() {
 
     mkdir build && cd build
 
-    cmake \
-        -DCMAKE_TOOLCHAIN_FILE="$FFBUILD_CMAKE_TOOLCHAIN" \
-        -DCMAKE_BUILD_TYPE=Release \
-        -DCMAKE_INSTALL_PREFIX="$FFBUILD_PREFIX" \
+    if [[ $TARGET == win* ]]; then
+        export CFLAGS="$CFLAGS -Dgettimeofday=ssh_gettimeofday"
+        export CXXFLAGS="$CFLAGS -Dgettimeofday=ssh_gettimeofday"
+    fi
+
+    cmake -GNinja -DCMAKE_TOOLCHAIN_FILE="$FFBUILD_CMAKE_TOOLCHAIN" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="$FFBUILD_PREFIX" \
         -DBUILD_SHARED_LIBS=OFF \
-        -DWITH_BLOWFISH_CIPHER=ON \
-        -DWITH_DSA=ON \
-        -DWITH_PCAP=ON \
-        -DWITH_SFTP=ON \
-        -DWITH_ZLIB=ON \
-        -DWITH_EXAMPLES=OFF \
-        -DWITH_SERVER=OFF \
-        -GNinja \
-        ..
+        -DWITH_EXAMPLES=OFF -DWITH_SERVER=OFF \
+        -DWITH_SFTP=ON -DWITH_ZLIB=ON ..
+
     ninja -j$(nproc)
     ninja install
 
     {
         echo "Requires.private: libssl libcrypto zlib"
-        echo "Cflags.private: -DLIBSSH_STATIC" 
+        echo "Cflags.private: -DLIBSSH_STATIC"
     } >> "$FFBUILD_PREFIX"/lib/pkgconfig/libssh.pc
 }
 
@@ -42,8 +38,4 @@ ffbuild_configure() {
 
 ffbuild_unconfigure() {
     echo --disable-libssh
-}
-
-ffbuild_ldflags() {
-    echo -Wl,--allow-multiple-definition
 }

--- a/scripts.d/50-libssh.sh
+++ b/scripts.d/50-libssh.sh
@@ -1,0 +1,49 @@
+#!/bin/bash
+
+SCRIPT_REPO="https://git.libssh.org/projects/libssh.git"
+SCRIPT_COMMIT="ac6d2fad4a8bf07277127736367e90387646363f"
+
+ffbuild_enabled() {
+    return 0
+}
+
+ffbuild_dockerbuild() {
+    git-mini-clone "$SCRIPT_REPO" "$SCRIPT_COMMIT" libssh
+    cd libssh
+
+    mkdir build && cd build
+
+    cmake \
+        -DCMAKE_TOOLCHAIN_FILE="$FFBUILD_CMAKE_TOOLCHAIN" \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX="$FFBUILD_PREFIX" \
+        -DBUILD_SHARED_LIBS=OFF \
+        -DWITH_BLOWFISH_CIPHER=ON \
+        -DWITH_DSA=ON \
+        -DWITH_PCAP=ON \
+        -DWITH_SFTP=ON \
+        -DWITH_ZLIB=ON \
+        -DWITH_EXAMPLES=OFF \
+        -DWITH_SERVER=OFF \
+        -GNinja \
+        ..
+    ninja -j$(nproc)
+    ninja install
+
+    {
+        echo "Requires.private: libssl libcrypto zlib"
+        echo "Cflags.private: -DLIBSSH_STATIC" 
+    } >> "$FFBUILD_PREFIX"/lib/pkgconfig/libssh.pc
+}
+
+ffbuild_configure() {
+    echo --enable-libssh
+}
+
+ffbuild_unconfigure() {
+    echo --disable-libssh
+}
+
+ffbuild_ldflags() {
+    echo -Wl,--allow-multiple-definition
+}

